projects/pt1/python/torch_mlir/jit_ir_importer/build_tools/abstract_interp_lib_gen.py

```diff
@@ -1702,13 +1708,25 @@ def aten〇clamp_min〡dtype(self_rank_dtype: Tuple[int, int], min: Union[int, f
         return torch.int64
     return self_dtype
 
+def aten〇clamp_min〇Tensor〡dtype(self_rank_dtype: Tuple[int, int], min_rank_dtype: Tuple[int, int]) -> int:
```

Comment:
drive by comment: all dtype functions should be tested. Should be a matter of adding something like

```python
@check_dtype_function(_check_tensors_with_the_same_dtype(num_of_tensors=2))
```

Same comment for dtype function below.